### PR TITLE
Core option for models more human readable

### PR DESCRIPTION
**Pull request by sonninnos**
Also removed the unknowns, because why not.

### CODE REVIEW

**Comment by rsn8887** [comment]
Awesome thanks. 

Can you add CBM2 also at the top? I don’t know how I missed that before.

File: libretro/libretro-core.c [hunk 1]
```diff
@@ -507,27 +507,27 @@ void retro_set_environment(retro_environment_t cb)
 #if  defined(__VIC20__)
       {
          "vice_vic20_model",
-         "VIC20 model; VIC20MODEL_VIC20_PAL|VIC20MODEL_VIC20_NTSC|VIC20MODEL_VIC21|VIC20MODEL_UNKNOWN",
+         "VIC20 model; VIC20 PAL|VIC20 NTSC|VIC21",
```
Comment (rsn8887) [inline libretro/libretro-core.c]: The model.h file shows in a comment that the vic21 model is the `SuperVIC (+16K)` can you change it? Thanks. I think the more accurate the model description are the better.

File: libretro/libretro-core.c [hunk 2]
```diff
@@ -507,27 +507,27 @@ void retro_set_environment(retro_environment_t cb)
...
       {
          "vice_c128_model",
-         "C128 model; C128MODEL_C128_PAL|C128MODEL_C128DCR_PAL|C128MODEL_C128_NTSC|C128MODEL_C128DCR_NTSC|C128MODEL_UNKNOWN",
+         "C128 model; C128 PAL|C128DCR PAL|C128 NTSC|C128DCR NTSC",
       },
 #elif  defined(__PET__)
       {
          "vice_pet_model",
-         "PET model; PETMODEL_2001|PETMODEL_3008|PETMODEL_3016|PETMODEL_3032|PETMODEL_3032B|PETMODEL_4016|PETMODEL_4032|PETMODEL_4032B|PETMODEL_8032|PETMODEL_8096|PETMODEL_8296|PETMODEL_SUPERPET|PETMODEL_UNKNOWN",
+         "PET model; 2001|3008|3016|3032|3032B|4016|4032|4032B|8032|8096|8296|SUPERPET",
```
Comment (rsn8887) [inline libretro/libretro-core.c]: Can you also add the CBM stuff after the PET stuff? Thanks!

**Comment by sonninnos** [comment]
Now ok?

**Comment by rsn8887** [comment]
Perfect thanks!

**Comment by twinaphex** [comment]
BTW, you guys should try updating to the new core options, so that you can add sublabels to the options.

**Pull request merged**